app/scripts/store/dimStoreItem.directive.js

```diff
@@ -67,11 +68,26 @@
           ngDialog.closeAll();
 
           if (!dimLoadoutService.dialogOpen) {
-            var bottom = ($(element).offset().top < 300) ? ' move-popup-bottom' : '';
-            var right = ((($('body').width() - $(element).offset().left - 320) < 0) ? ' move-popup-right' : '');
+
+            var settings = dimSettingsService.getSettings().$$state.value,
```

Comment:
@SunburnedGoose I'll be honest, I only did it this way because I couldn't find an example of appropriate use of your settings service.

Where should I look for an example of how you'd like it used?